services/minespace-web/src/App.js

```diff
@@ -1,31 +1,42 @@
 import React, { Fragment, Component } from "react";
-import { compose } from "redux";
-import { bindActionCreators } from "redux";
+import { compose, bindActionCreators } from "redux";
 import { connect } from "react-redux";
 import { BrowserRouter } from "react-router-dom";
 // eslint-disable-next-line
 import { hot } from "react-hot-loader";
 import { LoadingOutlined } from "@ant-design/icons";
 import { Layout, BackTop, Row, Col, Spin } from "antd";
 import { loadBulkStaticContent } from "@common/actionCreators/staticContentActionCreator";
-import { isAuthenticated } from "@/selectors/authenticationSelectors";
 import { getStaticContentLoadingIsComplete } from "@common/selectors/staticContentSelectors";
 import MediaQuery from "react-responsive";
-import Routes from "./routes/Routes";
+import * as PropTypes from "prop-types";
+import { isAuthenticated } from "@/selectors/authenticationSelectors";
 import { Header } from "@/components/layout/Header";
 import { Footer } from "@/components/layout/Footer";
 import ModalWrapper from "@/components/common/wrappers/ModalWrapper";
 import DocumentViewer from "@/components/syncfusion/DocumentViewer";
 import AuthenticationGuard from "@/HOC/AuthenticationGuard";
 import WarningBanner from "@/components/common/WarningBanner";
 import { detectIE } from "@/utils/environmentUtils";
+import Routes from "./routes/Routes";
 import configureStore from "./store/configureStore";
 import { MatomoLinkTracing } from "../common/utils/trackers";
 
 export const store = configureStore();
 
 Spin.setDefaultIndicator(<LoadingOutlined style={{ fontSize: 40 }} />);
 
+const propTypes = {
+  loadBulkStaticContent: PropTypes.func.isRequired,
+  isAuthenticated: PropTypes.bool,
+  staticContentLoadingIsComplete: PropTypes.bool,
+};
+
+const defaultProps = {
+  isAuthenticated: false,
+  staticContentLoadingIsComplete: false,
```

Comment:
Is this one necessary?

Reply:
Without it, it breaks the current eslint rules:

`ESLint: propType "isAuthenticated" is not required, but has no corresponding defaultProps declaration.(react/require-default-props)`